src/Cake.Common/Tools/MSBuild/MSBuildVerbosityExtensions.cs

```diff
@@ -35,5 +35,29 @@ public static string GetMSBuildVerbosityName(this Verbosity verbosity)
                     throw new CakeException("Encountered unknown MSBuild build log verbosity.");
             }
         }
+
+        /// <summary>
+        /// Gets the MSBuild <see cref="Verbosity"/> from string value.
+        /// </summary>
+        /// <param name="verbosity">The verbosity string value.</param>
```

Comment:
Can we add a `Remarks` section here with the valid verbosity values, so they appear in the documentation.

Reply:
I added description to remarks and exception message.